src/components/ProjectView/ImageCarousel.vue

```diff
@@ -58,61 +100,128 @@ const getImageSrc = (src) => {
       v-for="(item, index) in data"
       :key="index"
     >
-      <div class="carousel__item">
+      <div class="selected__item">
         <img
           :src="getImageSrc(largeImageSrc)"
-          class="div__item"
+          class="selected__image"
           :alt="item.OBJECT"
         >
       </div>
     </Slide>
   </Carousel>
-
-  <!-- Thumbnails Carousel -->
-  <Carousel
+  <div
```

Comment:
If I'm reading this right the carousel is now a custom component?

Reply:
correct